.appveyor.yml

```diff
@@ -68,5 +68,5 @@ build_script:
 
       pushd . && cd tests && go mod tidy && go mod download && popd
 
-      if defined APPVEYOR_REPO_TAG_NAME ( dotnet msbuild /t:ReleaseProcess /v:Detailed build.proj )
+      echo %APPVEYOR_REPO_TAG_NAME%|findstr "^v[0-9]" >Nul 2>&1 && ( dotnet msbuild /t:ReleaseProcess /v:Detailed build.proj ) || exit 0
```

Comment:
Does the `|| exit 0` suppress real failures from the `dotnet msbuild`?

Reply:
Good point. I updated this and tested it on the windows VM. Had to convert it into a script as appveyor doesn't support multiline CMD.